rllib/connectors/learner/add_columns_from_episodes_to_train_batch.py

```diff
@@ -100,6 +100,8 @@ def __call__(
                     Columns.TERMINATEDS,
                     items_to_add=(
                         [False] * (len(sa_episode) - 1) + [sa_episode.is_terminated]
+                        if len(sa_episode) > 0
```

Comment:
We should probably remove any zero-length episodes in the loop here. They cannot be used for learning anyways.

Reply:
Leave this to @sven1977 to decide.

Reply:
@sven1977 could we give this a go? I think this is fine like this. The other option is to remove zero-length epiosdes entirely as they offer nothing to learn from. This could also happen if an agent in a multi-agent scenario received only the initial observation and nothing else.